web/src/components/Plugin/PluginPage.tsx

```diff
@@ -198,7 +222,13 @@ const PluginPage: React.FC<Props> = ({
                 pluginList.filter(
```

Comment:
```suggestion
                (isEnablePlugin ? enablePluginList : pluginList).filter(
```
Actually, another good idea is that we have a variable for enablePluginList